docker/lr-pb/detect_run_info.py

```diff
@@ -59,7 +59,7 @@
 
     # First, populate run info (ri) hash with information gleaned from run metadata
     for blob in blobs:
-        if blob.name.endswith(".bam") and not blob.name.endswith(".scraps.bam"):
+        if blob.name.endswith(".subreads.bam") and not blob.name.endswith(".scraps.bam"):
```

Comment:
I wonder if a better solution would be to only grab the bam files in the folder specified (and not in all subfolders).

I think the issue was that it's going into the subfolders to grab the other bam files.  

Do we want / need to look at all subdirectories to find the bam files there?

Reply:
Yeah, I've wondered this myself.  I suppose in theory we could be given a directory of N flowcells all for a single sample and want to process them all together, in which case recursing through the subdirectories is not a bad idea.  But I'm not sure if that use case has ever *actually* come up yet.

And looking for ".subreads.bam" *is* technically what we want here, right?  If we were to accept any BAM, we'd have to do a lot more checking as to what the contents of those BAMs represent, since it wouldn't make sense to run the CCS task on already-CCS'd data.